sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -2493,5 +2500,6 @@ if used in application code.
 |2|2023-08-01|Pablo Reble, Ewan Crawford, Ben Tracy, Julian Miller,
 Maxime France-Pillois
 |Promote status to experimental
+|3|2025-05-09|Pablo Reble|Update format
```

Comment:
Why rev the spec on an editorial change?

Reply:
Discussed offline, editorial changes won't qualify for a new revision. Will bump version in a follow up PR.